receiver/ciscoosreceiver/README.md

```diff
@@ -0,0 +1,92 @@
+| Status        |           |
+| ------------- |-----------|
+| Stability     | [development]: metrics   |
+| Distributions | [] |
+| Issues        | [![Open issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aopen%20label%3Areceiver%2Fciscoos%20&label=open&color=orange&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aopen+is%3Aissue+label%3Areceiver%2Fciscoos) [![Closed issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aclosed%20label%3Areceiver%2Fciscoos%20&label=closed&color=blue&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aclosed+is%3Aissue+label%3Areceiver%2Fciscoos) |
+| Code coverage | [![codecov](https://codecov.io/github/open-telemetry/opentelemetry-collector-contrib/graph/main/badge.svg?component=receiver_ciscoosreceiver)](https://app.codecov.io/gh/open-telemetry/opentelemetry-collector-contrib/tree/main/?components%5B0%5D=receiver_ciscoosreceiver&displayType=list) |
+| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | [@dmitryax](https://www.github.com/dmitryax) |
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/component-stability.md#development
+<!-- end autogenerated section -->
+
+The Cisco OS Receiver is a modular receiver that collects metrics from Cisco network devices via SSH connections. It supports multiple scrapers for comprehensive network monitoring and provides native OpenTelemetry OTLP metrics generation.
+
+## Configuration
+
+The following settings are available:
+
+| Setting | Type | Required | Description |
+|---------|------|----------|-------------|
+| `devices` | []DeviceConfig | Yes | List of Cisco devices to monitor |
+| `collection_interval` | duration | No | How often to collect metrics (default: 60s) |
+| `timeout` | duration | No | SSH connection and command timeout (default: 30s) |
+| `scrapers` | ScrapersConfig | Yes | Enable/disable specific scrapers |
+
+### Device Configuration
+
+| Setting | Type | Required | Description |
+|---------|------|----------|-------------|
+| `host` | string | Yes | Device address in `host:port` format |
+| `username` | string | Yes | SSH username for authentication |
+| `password` | string | No* | Password for authentication |
+| `key_file` | string | No* | Path to SSH private key file |
+
+*Either `password` or `key_file` is required, but not both.
+
+### Scrapers Configuration
+
+| Setting | Type | Description |
+|---------|------|-------------|
+| `bgp` | bool | BGP session metrics |
+| `environment` | bool | Temperature and power metrics |
+| `facts` | bool | System information metrics |
+| `interfaces` | bool | Interface status and statistics |
+| `optics` | bool | Optical transceiver metrics |
```

Comment:
Do we want to potentially have configurations under particular scraper and have metrics defined per scraper? If that the case, I guess we should follow the same configuration interface as in [the host metrics receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/hostmetricsreceiver) where every scraper is a group, not boolean